torchtext/experimental/datasets/language_modeling.py

```diff
@@ -65,63 +62,31 @@ def get_vocab(self):
         return self.vocab
 
 
-def _get_datafile_path(key, extracted_files):
-    for fname in extracted_files:
-        if key in fname:
-            return fname
-
-
-def _setup_datasets(dataset_name, tokenizer=get_tokenizer("basic_english"),
-                    root='.data', vocab=None, removed_tokens=[],
-                    data_select=('train', 'test', 'valid')):
+def _setup_datasets(dataset_name, root='.data', vocab=None,
```

Comment:
Why do we want to change the order of `tokenizer` and `vocab`? This will be a BC breaking.